spyder/config/main.py

```diff
@@ -94,7 +94,6 @@
               'use_custom_margin': True,
               'custom_margin': 0,
               'use_custom_cursor_blinking': False,
-              'custom_cursor_blinking': 1000,

```

Comment:
@goanpeca, there is no need to do that because values here are placed here only if we plan to change/update them in the future.